decentraland/unity-explorer
# fix: floating rocks puzzle scene / moving platforms scene

## What does this PR change?

Fixes #1385 
Fixes #1111 

- Adds the `SDKTransform` to `CAMERA_ENTITY` as we did with `PLAYER_ENTITY`. If not, any GetMutable() call from SDK will break the scene flow.
- Removes the `CachedTween` from the TweenComponent. We will only rely on the `isDirty` flag
- Changes the order in which we setup the tween component. We first setup the transform with start values, and then we set the tween value with the transform values, in this case they should both be aligned. Setting it the other way around may the rotation go backwards. Probably because there was an incomplete delta from previous rotation, and therefore the shortest path algo made it go backwards.
- Removes the `[ThrottlingEnabled]` tag from the Tween systems. They should not be throttled, otherwise we will have jumpy animations
- Resorts orders for Tween systems
- After resorting, I detected a bug that we were overwriting scene transform fields not related to field. Therefore, I changed how we write the result

## How to test the changes?

Check prod parity with

1. Go to scene `(78,-7)` (Rotating Platforms)
2. Go to scene `(84,-4)` (Floating rocks)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
